raster/rt_core/rt_raster.c

```diff
@@ -368,7 +368,7 @@ rt_raster_set_srid(rt_raster raster, int32_t srid) {
 	_rt_raster_geotransform_warn_offline_band(raster);
 }
 
-int
+uint16_t
```

Comment:
why 16?

Reply:
`numBands` is a uint16_t and `rt_raster_get_width` and `rt_raster_get_height` were already returning a 16 for the similar reasons.